ivy/functional/ivy/sorting.py

```diff
@@ -234,6 +234,27 @@ def sort(
     )
 
 
+@to_native_arrays_and_back
+@handle_out_argument
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+def lexsort(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    axis: int = -1,
+    descending: bool = False,
+    stable: bool = True,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
```

Comment:
just to confirm, does the contribution guide or the todo list issue mension that you need to filling the docstring for the function as well?

Reply:
I didn't because docstring was not requested. I saw a separate tasks for docstrings